markdown/about.md

```diff
@@ -21,6 +21,7 @@ The core team members who administer the nf-core project are listed below:
 - [@MaxUlysse](https://github.com/MaxUlysse): Maxime Garcia
 - [@mirpedrol](https://github.com/mirpedrol): Júlia Mir Pedrol
 - [@mribeirodantas](https://github.com/mribeirodantas): Marcel Ribeiro-Dantas
+- [@FranBonath](https://github.com/FranBonath): Franziska Bonath
```

Comment:
I think names are alphabetically ordered, so you can go 🆙 🙂

Reply:
Hm, I fail to see that order. Harshil and Phil would be way too far up in that case. Do we want it to be alphabetical?

Reply:
I think it used to be by last name, but somehow it got messed up (or maybe some people sorted by first and some by last? 😆

Reply:
Isn't it by github ID?

Reply:
I think you are right @mirpedrol 😆. Ok, I'll correct it then.